nuke-from-orbit/loadtester

```diff
@@ -0,0 +1,372 @@
+   --project $gcp_project_id \
+   --zone $gcp_zone \
+   --scopes "https://www.googleapis.com/auth/cloud-platform" \
+   --num-nodes "$gcp_cluster_node_count" \
+   --machine-type $gcp_cluster_machine_type \
+   --cluster-version "1.16.11-gke.5"
+}
+
+gke_loadtest_cluster_ip_gcloud() {
+  gcloud compute addresses create ${loadtest_name} --global
+  loadtest_ip_address=$(gcloud compute addresses describe  $loadtest_name --global --format json | jq -r '.address')
+}
+
+gke_loadtest_cluster_destroy_gcloud() {
+  gcloud container clusters delete $loadtest_name --zone $gcp_zone --quiet
+}
+
+gke_loadtest_cluster_ip_destroy_gcloud() {
+  gcloud compute addresses delete $loadtest_name --global --quiet
+}
+
+# Parse kubernetes jinja templates into proper config files
+parse_config() {
+  echo "Cleaning out any old configs"
+  rm ${script_dir}/kubernetes-config/*.yaml
+  echo "Parsing kubernetes config templates"
+  if [ -z $setup_command ]
+  then
+    python ${script_dir}/kubernetes-config/parse_kube_templates.py --only-user-config --image-tag $image_tag
+  else
+    python ${script_dir}/kubernetes-config/parse_kube_templates.py --image-tag $image_tag
+  fi
+}
+
+# Self-contained only: Provision Looker instance
+provision_looker() {
+  echo "Provisioning Looker instance"
+  python ${script_dir}/looker-provisioning/looker_setup.py
+  echo "Provisioning attempt complete."
+}
+
+# Parse relevant config file to set needed variables
+set_variables() {
+  gcp_project_id=$(cat $config_file | jq -r '.gcp_project_id // empty')
+  loadtest_name=$(cat $config_file | jq -r '.loadtest_name // empty')
+  loadtest_dns_domain=$(cat $config_file | jq -r '.loadtest_dns_domain // empty')
+  gcp_oauth_client_id=$(cat $config_file | jq -r '.gcp_oauth_client_id // empty')
+  gcp_oauth_client_secret=$(cat $config_file | jq -r '.gcp_oauth_client_secret // empty')
+  gcp_zone=$(cat $config_file | jq -r '.gcp_zone // empty')
+  gcp_iap_email=$(cat $config_file | jq -r '.gcp_iap_email // empty')
+  gcp_cluster_node_count=$(cat $config_file | jq -r '.gcp_cluster_node_count // empty')
+  gcp_cluster_machine_type=$(cat $config_file | jq -r '.gcp_cluster_machine_type // empty')
+  looker_user=$(cat $config_file | jq -r '.looker_user // empty')
+  looker_pass=$(cat $config_file | jq -r '.looker_pass // empty')
+  aws_access_key=$(cat $config_file | jq -r '.aws_access_key // empty')
+  aws_secret_key=$(cat $config_file | jq -r '.aws_secret_access_key // empty')
+  aws_session_token=$(cat $config_file | jq -r '.aws_session_token // empty')
+  lookml_project_repo=$(cat $config_file | jq -r '.lookml_project_repo // empty')
+
+  if [ -z $gcp_oauth_client_id ]; then echo "Could not find gcp_oauth_client_id"; exit 1; fi
+  if [ -z $gcp_oauth_client_secret ]; then echo "Could not find gcp_oauth_client_secret"; exit 1; fi
+  if [ -z $gcp_iap_email ]; then echo "Could not find gcp_iap_email"; exit 1; fi
+
+  if ! [ $setup_command == 'self-contained' ]
```

Comment:
== should probably be -eq

Reply:
-eq is only for numbers apparently. However, the POSIX compliant version is just `=`.  Bash is weird.